fedmsg.d/fas_credentials.py

```diff
@@ -1,8 +1,6 @@
-import getpass
```

Comment:
This broke running the tests from the repo's root directory since it'd trigger a password prompt.